src/GltfView/gltf_view.js

```diff
@@ -0,0 +1,82 @@
+import { GltfState } from '../GltfState/gltf_state.js';
+import { gltfRenderer } from '../renderer';
+import { gltfWebGl } from '../webgl';
+
+class GltfView
+{
+    constructor(canvas)
+    {
+        this.canvas = canvas;
+        this.context = this.canvas.getContext("webgl2", { alpha: false, antialias: true });
+        this.webGl = new gltfWebGl();
+        this.webGl.context = this.context;
+        this.renderer = new gltfRenderer(this.webGl);
+    }
+
+    createState()
+    {
+        return new GltfState();
+    }
+
+    renderFrameToCanvas(state)
+    {
+        this.renderer.clearFrame(state.renderingParameters.clearColor);
+
+        // TODO: this should probably not be done here
```

Comment:
Resolve todo?

Reply:
i think at the moment this is necessary there. We might need to revisit this in the google test suite integration stage